### DISCUSSION TITLE
closeIcon

### DISCUSSION THREAD

**Discussion by sorenjensen** [MurhafSousli/ngx-gallery#496] [Q&A] [answered]
how do I specify svg data for custom closeicon ? tried adding svg source but cant get it to show other than the default close icon

**Comment by MurhafSousli** [comment, marked as answer]
It's an option of `LightboxConfig` https://github.com/MurhafSousli/ngx-gallery/wiki/Lightbox-API, use the same way you would override global config for `GalleryConfig`

```ts
LightboxModule.withConfig({
  closeIcon: YourSVGStringValue
})
```
